src/pages/[platform]/how-amplify-works/faq/index.mdx

```diff
@@ -29,42 +29,43 @@ export function getStaticProps(context) {
 
 }
 
-**In Gen 1, Amplify offered a set of use case categories for building applications (for example, Authentication, Analytics, API, DataStore, Geo, and Predictions). Are those same categories available in Gen 2?**
 
-In the launch of public preview, Amplify code-first DX (Gen 2) supports Auth and Data. Other use cases can be implemented using AWS Cloud Development Kit (AWS CDK) constructs.
+**Is Gen 2 backward compatible with existing Amplify projects on Gen 1?**
 
-**Does Amplify Gen 2 support DataStore?**
+No, Gen 2 is not backward compatible with existing Amplify projects on Gen 1. Amplify Gen 1 and Gen 2 follow different architectural and tooling paradigms in order for Gen 2 to address common feedback customers had about Gen 1. The two versions are not compatible.
```

Comment:
I'm not a fan of the way this is answered because it leaves a lot of room for interpretation as to what we mean with "backwards compatible"

Reply:
Maybe something like, `Can I combine a Gen 2 project with Gen 1?` or `Can I add add additional functionality using Gen 2 to my existing Gen 1 project?` ?

Reply:
**If I have a Gen 1 app, can I use Gen 2 with it?**

Amplify Gen 1 and Gen 2 follow different architectural and tooling paradigms, which was necessary to address  common customer feedback from Gen 1. You will need to use our upcoming migration tooling to move from a Gen 1 to Gen 2 app. You cannot use Amplify Gen 1 (Studio/CLI) in the same app as Gen 2.